src/components/Event/EventBox.tsx

```diff
@@ -0,0 +1,79 @@
+export interface EventBoxProps {
+  EventName: string;
+  Walkin: boolean;
+  MaxEnrollment: number;
+  ThaiFaculty: string;
+  EngFaculty: string;
+  Time: number;
+  Location: string;
+  Thai: boolean;
+}
+const EnrollBox: React.FC<{
+  Walkin: boolean;
+  MaxEnrollment: number;
+}> = ({ Walkin, MaxEnrollment }) => (
+  <div className="mb-2 flex flex-col items-start gap-3 text-white">
+    <div className="rounded-full bg-pink-500 px-2 py-1 ">
+      <p className="text-xs">
+        จำนวนที่นั่งต่อรอบ / Capacity per session : {MaxEnrollment}
+      </p>
+    </div>
+    {Walkin && (
+      <div className="flex rounded-full border-2 border-white px-2 py-1">
+        <p className="text-xs">Walk-in registration only</p>
+      </div>
+    )}
+  </div>
+);
+
+const EventBox: React.FC<EventBoxProps> = ({
+  EventName,
+  Walkin,
+  MaxEnrollment,
+  ThaiFaculty,
+  EngFaculty,
+  Time,
+  Location,
+  Thai,
+}) => {
+  const SessionTime = ` ${Time > 60 ? Time / 60 : Time} ${
+    Time > 60
+      ? "ชั่วโมง ต่อรอบ / hours per session"
+      : "นาที ต่อรอบ/ mins per session"
+  }`;
+  return (
+    <div className="box-border w-72 rounded-lg border-2 border-white bg-pink-500 shadow-md ring-2 ring-white ring-opacity-5">
+      <a href="/">
+        <div className="border-1 rounded-md border-black bg-gradient-to-r from-[#393570]  to-[#CA4072] px-4 py-2">
```

Comment:
Arbitrary values here is bad practice. We already have these colors in tailwind.config

```suggestion
        <div className="border-1 rounded-md border-black bg-gradient-to-r from-indigo-900  to-pink-550 px-4 py-2">
```